arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTraceViewController.swift

```diff
@@ -0,0 +1,391 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ConfigureSubnetworkTraceViewController: UITableViewController {
+    // Reference to the switches.
+    @IBOutlet weak var barriersSwitch: UISwitch!
+    @IBOutlet weak var containersSwitch: UISwitch!
+    // References to interactable cells.
+    @IBOutlet weak var attributesCell: UITableViewCell!
+    @IBOutlet weak var comparisonCell: UITableViewCell!
+    @IBOutlet weak var valueCell: UITableViewCell!
+    @IBOutlet weak var addConditionLabel: UILabel!
+    @IBOutlet weak var resetLabel: UILabel!
+    @IBOutlet weak var traceLabel: UILabel!
+    // References to labels.
+    @IBOutlet weak var attributeLabel: UILabel!
+    @IBOutlet weak var comparisonLabel: UILabel!
+    @IBOutlet weak var valueLabel: UILabel!
+    // References to cells that act as buttons.
+    @IBOutlet weak var addConditionButton: UITableViewCell!
+    @IBOutlet weak var resetButton: UITableViewCell!
+    @IBOutlet weak var traceButton: UITableViewCell!
+    // Reference to the text view.
+    @IBOutlet weak var textView: UITextView!
+    // References to the switch actions.
+    @IBAction func barriersSwitchAction(_ sender: UISwitch) {
+        sourceTier?.traceConfiguration?.includeBarriers = sender.isOn
+    }
+    @IBAction func containersSwitchAction(_ sender: UISwitch) {
+        sourceTier?.traceConfiguration?.includeContainers = sender.isOn
+    }
+    // Feature service for an electric utility network in Naperville, Illinois.
+    let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
+    // For creating the default starting location.
+    let deviceTableName = "Electric Distribution Device"
+    let assetGroupName = "Circuit Breaker"
+    let assetTypeName = "Three Phase"
+    let globalID = UUID(uuidString: "1CAF7740-0BF4-4113-8DB2-654E18800028")
+    // For creating the default trace configuration.
+    let domainNetworkName = "ElectricDistribution"
+    let tierName = "Medium Voltage Radial"
+    // An array of the types of AGSUtilityAttributeComparisonOperators as strings.
+    let comparisonsStrings = ["Equal", "NotEqual", "GreaterThan", "GreaterThanEqual", "LessThan", "LessThanEqual", "IncludesTheValues", "DoesNotIncludeTheValues", "IncludesAny", "DoesNotIncludeAny"]
+    // An array of the types of AGSUtilityAttributeComparisonOperators.
+    let comparisons: [AGSUtilityAttributeComparisonOperator] = [.equal, .notEqual, .greaterThan, .greaterThanEqual, .lessThan, .lessThanEqual, .includesTheValues, .doesNotIncludeTheValues, .includesAny, .doesNotIncludeAny]
```

Comment:
Might be better to have a dictionary than 2 arrays?

Reply:
I originally attempted to use a dictionary but it later caused me more trouble to obtain its keys/values as strings. Please let me know if you have an alternate solution!